controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -392,7 +561,13 @@ func (r *StorageConsumerReconciler) reconcileCephClientHealthChecker() error {
 	r.updateCephResourceStatus(cephClientHealthCheckerDetails)
 
 	if err != nil {
-		r.Log.Error(err, "Failed to update CephClient.", "CephClient", klog.KRef(r.cephClientHealthChecker.Namespace, r.cephClientHealthChecker.Name))
+		r.Log.Error(
+			err,
+			"Failed to update CephClient.",
+			"CephClient",
+			klog.KRef(r.cephClientHealthChecker.Namespace,
+				r.cephClientHealthChecker.Name),
```

Comment:
See comment above about breaking lines

Reply:
Health checker changes will be done in a seperate PR, just keeping only the cephfs changes